packages/data-context/src/util/hasTypescript.ts

```diff
@@ -1,5 +1,11 @@
 export function hasTypeScriptInstalled (projectRoot: string) {
   try {
+    // mocking this module is fairly difficult under unit test. We need to mock this for the ProjectConfigIpc unit tests
+    // as the scaffolded projects in the data-context package do not install dependencies related to the project.
+    if (process.env.CYPRESS_INTERNAL_MOCK_TYPESCRIPT_INSTALL === 'true') {
```

Comment:
I couldn't figure out another way to test this. Not great, but at least gets us what we need